README.md

```diff
@@ -82,5 +113,5 @@ GET /files/a123b1234
 
 ##Put a file
 ````
-PUT /files/a123b1234
+PUT /files/a123b1234?length=nn&offset=nn
```

Comment:
Maybe use the Content-Range header for this instead? http://www.w3.org/Protocols/rfc2616/rfc2616-sec14.html (14.16)